osu.Game.Rulesets.Mania/Difficulty/ManiaPerformanceCalculator.cs

```diff
@@ -105,7 +105,7 @@ private double computeStrainValue()
 
         private double computeAccuracyValue(double strainValue)
         {
-            double hitWindowGreat = (Beatmap.HitObjects.First().HitWindows.Great / 2 - 0.5) / TimeRate;
```

Comment:
have you double-checked this against stable and made sure this is correct?

Reply:
On a double check, headless and master differ in this. Headless has the 0.5 offsets, so we may want to keep them...